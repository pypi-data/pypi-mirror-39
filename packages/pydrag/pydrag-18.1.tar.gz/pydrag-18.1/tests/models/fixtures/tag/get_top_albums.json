{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=2&method=tag.getTopAlbums&page=1&tag=rap"
            },
            "response": {
                "body": {
                    "string": "{\"albums\":{\"album\":[{\"name\":\"The Eminem Show\",\"mbid\":\"CENSORED3bd6\",\"url\":\"https://www.last.fm/music/Eminem\",\"artist\":{\"name\":\"Eminem\",\"mbid\":\"CENSORED13d4\",\"url\":\"https://www.last.fm/music/Eminem\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"1\"}},{\"name\":\"Views\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Drake\",\"artist\":{\"name\":\"Drake\",\"mbid\":\"CENSORED69a6\",\"url\":\"https://www.last.fm/music/Drake\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"2\"}}],\"@attr\":{\"tag\":\"rap\",\"page\":\"1\",\"perPage\":\"2\",\"totalPages\":\"6769\",\"total\":\"13538\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "1444"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:53 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
