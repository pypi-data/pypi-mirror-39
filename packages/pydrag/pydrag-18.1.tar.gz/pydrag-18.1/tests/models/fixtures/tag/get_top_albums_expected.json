{
    "data": [
        {
            "artist": {
                "mbid": "CENSORED13d4",
                "name": "Eminem",
                "url": "https://www.last.fm/music/Eminem"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSORED3bd6",
            "name": "The Eminem Show",
            "rank": 1,
            "url": "https://www.last.fm/music/Eminem"
        },
        {
            "artist": {
                "mbid": "CENSORED69a6",
                "name": "Drake",
                "url": "https://www.last.fm/music/Drake"
            },
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "",
            "name": "Views",
            "rank": 2,
            "url": "https://www.last.fm/music/Drake"
        }
    ],
    "limit": 2,
    "page": 1,
    "tag": "rap",
    "total": 13538
}
